tests/kernel/run_ltp.pm

```diff
@@ -11,15 +11,30 @@ use 5.018;
 use warnings;
 use base 'opensusebasetest';
 use testapi qw(is_serial_terminal :DEFAULT);
+use serial_terminal 'select_serial_terminal';
+use power_action_utils 'power_action';
 use utils;
 use version_utils 'is_sle';
 use Time::HiRes qw(clock_gettime CLOCK_MONOTONIC);
+use Utils::Backends qw(is_pvm);
 use serial_terminal;
 use Mojo::File 'path';
 use Mojo::JSON;
+use LTP::utils 'prepare_ltp_env';
 use LTP::WhiteList;
 require bmwqemu;
 
+sub do_reboot {
+    my $self = shift;
+
+    record_info("reboot");
+    power_action('reboot', textmode => 1, keepconsole => is_pvm);
+    reconnect_mgmt_console if is_pvm;
```

Comment:
`reconnect_mgmt_console` is also needed in baremetal tests: `get_var('LTP_BAREMETAL')`

Reply:
Thanks fixed.